### PR TITLE
spdlog 1.13.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre># spdlog Release Notes

### Enhancements
* Qt Sink: Allow for darker colors for light backgrounds (#2817). Thanks to @krawq.
* Set `CMAKE_BUILD_TYPE` only for single-config generators (#2839). Thanks to @moritz-h.
* Provide `spdlog_header_only` in build directory export (#2846). Thanks to @robertmaynard.
* Add `SPDLOG_TO_VERSION` to compare spdlog version (#2853). Thanks to @kegechen.
* Support `async_overflow_policy::discard_new` (#2876). Thanks to @walkerlala.
* Make `syslog_sink.h`'s `syslog_prio_from_level` protected (#2918). Thanks to @shannonbooth.
* Fix OS availability check of `pthread_threadid_np` for iOS (#2897). Thanks to @lowdesertpunk.
* Add `.git-blame-ignore-revs` to ignore clang-format related commits (#2899). Thanks to @lowdesertpunk.
* Match `SPDLOG_CONSTEXPR_FUNC` to `FMT_CONSTEXPR` (#2901). Thanks to @kkraus14.

### Bug Fixes
* Check `fd_` is not nullptr in `file_helper` to prevent possible crash if disk is full  (#2972).
* Fix encoding issue in qt_sinks (#2862). Thanks to @neothenil and @tt4g .
* Fix wrong thread_id (TID) in `systemd_sink.h` (#2919). Thanks to @M010.
* Update `example.cpp` to fix the vector issue in `bin_example` (#2963). Thanks to @zhuzhzh.
* Fix MSVC compile flag for no exceptions (#2974). Thanks to @Programicus.
* Fix bug in `circular_q::size()` (#2820).
* Fix link to fmt license file (#2967).

### Maintenance
* Added and updated various tests for `circular_q`.
* Removed obsolete part from CMake configuration files (#2871). Thanks to @albert-github.
* Removed `policy_max` from `cmake_minimum_required(..)`.
* Updated `clang-format` to google style and updated various formatting scripts.
* Bumped to catch2 v3.5.0.

### Contributors
Thanks to all the contributors who helped in this release: @krawq, @xvitaly, @moritz-h, @robertmaynard, @kegechen, @tt4g, @albert-github, @neothenil, @walkerlala, @shannonbooth, @M010, @marcusmueller, @lowdesertpunk, @zhuzhzh, and @Programicus.

Special thanks to @tt4g for helping with community questions and issues.
</pre>
</details>

### CODE REVIEW

**Comment by chenrui333** [comment]
> Needs some rev bumps; but one of these formulas depends on `boost` which is currently in the CI. Better wait to see if this'll be merged.

I can add `no long build conflict` for the other PR so that we can still get this PR going.

**Comment by chenrui333** [comment]
as some main concern would be the ABI compatibility with `spdlog 1.13.0`

**Review by p-linnane — APPROVED** [review]

**Comment by chenrui333** [comment]
there are no sonoma bottles for arrayfre and khiva

and `cryfs` need a bump for linux build

**Comment by chenrui333** [comment]
still some build failures.

**Comment by chenrui333** [comment]
reopen to see where we are with this PR

File: Formula/g/glibmm.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Glibmm < Formula
   url "https://download.gnome.org/sources/glibmm/2.78/glibmm-2.78.1.tar.xz"
   sha256 "f473f2975d26c3409e112ed11ed36406fb3843fa975df575c22d4cb843085f61"
   license "LGPL-2.1-or-later"
+  revision 1
```
Comment (cho-m) [inline Formula/g/glibmm.rb:7]: No idea why, but rebuilding locally updated function parameters (`long long --> long`)
```console
❯ nm -gU /opt/homebrew/opt/glibmm@2.66/lib/libglibmm-2.4.dylib | rg create_now_local
0000000000006f88 T __ZN4Glib8DateTime16create_now_localERKNS_7TimeValE
0000000000006ef8 T __ZN4Glib8DateTime16create_now_localEv
0000000000006f40 T __ZN4Glib8DateTime16create_now_localEx

❯ nm -gU /opt/homebrew/opt/glibmm@2.66/lib/libglibmm-2.4.dylib | rg create_now_local | cut -d' ' -f3 | c++filt
Glib::DateTime::create_now_local(Glib::TimeVal const&)
Glib::DateTime::create_now_local()
Glib::DateTime::create_now_local(long long)
```

```console
❯ nm -gU /opt/homebrew/opt/glibmm@2.66/lib/libglibmm-2.4.dylib | rg create_now_local
0000000000006d48 T __ZN4Glib8DateTime16create_now_localERKNS_7TimeValE
0000000000006d00 T __ZN4Glib8DateTime16create_now_localEl
0000000000006cb8 T __ZN4Glib8DateTime16create_now_localEv

❯ nm -gU /opt/homebrew/opt/glibmm@2.66/lib/libglibmm-2.4.dylib | rg create_now_local | cut -d' ' -f3 | c++filt
Glib::DateTime::create_now_local(Glib::TimeVal const&)
Glib::DateTime::create_now_local(long)
Glib::DateTime::create_now_local()
```

Comment (cho-m) [reply Formula/g/glibmm.rb:7]: Actually rev bump seens to be done on `glibmm@2.66`.

Comment (chenrui333) [reply Formula/g/glibmm.rb]: 👍 💯

**Review by chenrui333 — APPROVED** [review]

**Comment by cho-m** [comment]
Looks like maybe one more revision bump (i.e. `libsigrok`). May try locally first to make sure rebuild actually fixes issue.

**Comment by cho-m** [comment]
Could alternatively merge PR if local attempt does fix issue and handle in follow up given usage for these aren't that high (and we have a JSON API issue that is essentially preventing upgrades for a lot of our user base right now).

`libsigrok`:
```
==> Analytics
install: 201 (30 days), 522 (90 days), 1,973 (365 days)
install-on-request: 121 (30 days), 301 (90 days), 1,117 (365 days)
build-error: 0 (30 days)
```

`sigrok-cli` (dependent of `libsigrok`):
```
==> Analytics
install: 119 (30 days), 322 (90 days), 1,077 (365 days)
install-on-request: 119 (30 days), 322 (90 days), 1,077 (365 days)
build-error: 0 (30 days)
```

**Comment by cho-m** [comment]
Ignoring missing `setuptools` dependency, `libsigrok` rebuild will fix issue. Going to trigger a merge and work on fix in a separate PR given our `long build` backlog.

**Review by cho-m — APPROVED** [review]

**Review by cho-m — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Comment by cho-m** [comment]
I think we had this issue before (approvals not working). CC @Homebrew/core who knows best way to resolve:

Screenshot 2024-03-19 at 9 53 02 PM

**Review by carlocab — APPROVED** [review]

**Review by fxcoudert — APPROVED** [review]

**Review by fxcoudert — APPROVED** [review]

**Review by ZhongRuoyu — APPROVED** [review]

**Pull request merged**